pom.xml

```diff
@@ -907,7 +907,7 @@
               <log4j.configuration>file:src/test/resources/log4j.properties</log4j.configuration>
               <log4j2.configurationFile>src/test/resources/log4j2-test.xml</log4j2.configurationFile>
               <java.io.tmpdir>${project.build.directory}/tmp</java.io.tmpdir>
-              <spark.driver.memory>4g</spark.driver.memory>
+              <spark.driver.memory>10g</spark.driver.memory>
```

Comment:
This is a nontrivial bump ... curious, why did we need it ?

Reply:
I was fighting with OOM in unit test, just the attempts to see if I can get rid of that